polygon/rest/base.py

```diff
@@ -31,7 +34,7 @@ def __init__(
             num_pools=num_pools,
             headers={
                 "Authorization": "Bearer " + self.API_KEY,
-                "User-Agent": "Python client",
+                "User-Agent": "Python client " + version,
```

Comment:
I'm not sure if this matters, but I feel like making it all lowercase and hyphens instead of spaces will make querying/indexing on this attribute easier down the road.

Reply:
@morningvera Do you have opinions here?